com.unity.ml-agents/Editor/BrainParametersDrawer.cs

```diff
@@ -185,11 +164,8 @@ static void DrawDiscreteVectorAction(Rect position, SerializedProperty property)
         /// <returns>The height of the drawer of the Vector Action.</returns>
         static float GetHeightDrawVectorAction(SerializedProperty property)
         {
-            var actionSize = 2 + property.FindPropertyRelative(k_ActionSizePropName).arraySize;
-            if (property.FindPropertyRelative(k_ActionTypePropName).enumValueIndex == 0)
-            {
-                actionSize += 1;
-            }
+            var actionSpecProperty = property.FindPropertyRelative(k_ActionSpecName);
+            var actionSize = 3 + actionSpecProperty.FindPropertyRelative("m_BranchSizes").arraySize;
```

Comment:
This is the opportunity to rename actionSize to numActionLines or something more descriptive